src/Loader.php

```diff
@@ -378,6 +401,10 @@ public static function load_scripts() {
 			return;
 		}
 
+		if ( ! current_user_can( static::get_analytics_capability() ) ) {
```

Comment:
Nitpick - what do you think about putting this in a helper function?

Reply:
> what do you think about putting this in a helper function?

Added in `82f59db2`